libcudacxx/include/cuda/std/detail/libcxx/include/__ranges/rend.h

```diff
@@ -0,0 +1,191 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+#ifndef _LIBCUDACXX___RANGES_REND_H
+#define _LIBCUDACXX___RANGES_REND_H
+
+#ifndef __cuda_std__
+#include <__config>
+#endif // __cuda_std__
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+#include "../__concepts/class_or_enum.h"
+#include "../__concepts/same_as.h"
+#include "../__iterator/concepts.h"
+#include "../__iterator/readable_traits.h"
+#include "../__iterator/reverse_iterator.h"
+#include "../__ranges/access.h"
+#include "../__ranges/rbegin.h"
+#include "../__type_traits/is_reference.h"
+#include "../__type_traits/remove_cvref.h"
+#include "../__type_traits/remove_reference.h"
+#include "../__utility/auto_cast.h"
+
+_LIBCUDACXX_BEGIN_NAMESPACE_RANGES
+
+#if _CCCL_STD_VER > 2014 && !defined(_LIBCUDACXX_COMPILER_MSVC_2017)
```

Comment:
question: in rbegin it's `_CCCL_STD_VER >= 2017`. Is there any convention you are using to choose between two schemes?

Reply:
Yes there is a convention by me to change all `>` or `<` to `>=` and `<=` because that shows clearly the standard mode that is expected. Unfortunately the codebase is highly inconsistent and sometimes it slips through